e2e/smartvol_ops_test.go

```diff
@@ -166,6 +206,20 @@ func testSmartVolumeDisperse(t *testing.T) {
 	r.Nil(brickSizeTest(volinfo.Subvols[0].Bricks[1].Path, 16, 21))
 	r.Nil(brickSizeTest(volinfo.Subvols[0].Bricks[2].Path, 16, 21))
 
+	expandReq := api.VolExpandReq{
```

Comment:
@vpandey-RH can you add some negative test cases also?

and also I see once `DistributeCount` is set for volume expansion in tests, can you please cover 
remaining fields in volume Expand struct like `Replica` , `bricks` etc.

Reply:
@Madhu-1  Brick Expand will consider only two options Distribute Count in req should be equal to volume's distribute count and size != 0.  Even if I provide other options it wont matter because they are not considered and the effect of these options have been tested in conventional volume-expand tests. 

I will add some negative tests as well.

Reply:
Done